### ISSUE TITLE
Keycloak integration

### ISSUE THREAD

**Issue by Peco602** [moloch--/RootTheBox#579]
Would it be possible to include Keycloak integration in addition to Azure AD?

**Comment by eljeffeg** [comment]
Yeah, I'd certainly like to expand the authentication options.  LDAP, SAML, OAUTH  I guess Keycloak facilitates such auth schemes.  Just a big time investment.

**Comment by cloud-blueteam** [comment]
+1 on SAML integration
